tox.ini

```diff
@@ -45,7 +45,7 @@ commands =
      docker rmi opsdroid-image:tmp
 
 [testenv:docs]
-basepython = python3
+basepython = python3.8
```

Comment:
I don't know if this is a good idea or not, but the build is broken with 3.9.

Reply:
We can always change it back if and when 3.9 doesn't break.